orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/cdp-token.xml.j2

```diff
@@ -23,6 +23,33 @@
          <name>Default</name>
          <enabled>true</enabled>
       </provider>
+
+{% if salt['pillar.get']('gateway:envAccessGroup') is defined and salt['pillar.get']('gateway:envAccessGroup') | length > 0 %}
+      <provider>
+         <role>identity-assertion</role>
+         <name>HadoopGroupProvider</name>
+         <enabled>true</enabled>
+         <param>
+            <name>CENTRAL_GROUP_CONFIG_PREFIX</name>
+            <value>gateway.group.config.</value>
+         </param>
+      </provider>
+
+      <provider>
+         <role>authorization</role>
+         <name>AclsAuthz</name>
```

Comment:
The CompositeAuthz provider was intended to be used here which will allow us to enable Ranger as well at some point. This shouldn't be considered a blocker but it needs to be understood that enabling Ranger in the gateway will just require some changes to this config - which would be needed anyway to add Ranger but there will be a little more as well.